src/yang/ietf-bgp-common.yang

```diff
@@ -308,6 +307,20 @@ submodule ietf-bgp-common {
       "Configuration parameters relating to the transport protocol
        used by the BGP session to the peer.";
 
+    leaf local-address {
+      type union {
+        type inet:ip-address;
+        type leafref {
+          path "../../../../interfaces/interface/name";
```

Comment:
Rather than using a relative path, let us use an absolute path by defining this as

'type if:interface-ref'

and importing the ietf-interfaces module in ietf-bgp-common.

Reply:
Changed.